api/src/lib/validator/mixed-validation.js

```diff
@@ -16,7 +16,7 @@ const mixedValidation = (joiSchema, customSchema) => {
 
       return Promise.all(
         Object.keys(customSchema).map((path) => {
-          if (! value[path]) {
+          if (!value || !value[path]) {
```

Comment:
If a request was made to an endpoint utilizing mixed validation with an empty payload, then the endpoint would simply timeout instead of providing a 400/500 response.  The return block on line 23 would not catch the undefined and would return a malformed promise that would be passed to `Promise.all`, where the catch block would try to call `next` but nothing would occur due to the malformation.  Simply rejecting any value that is falsy prevents the error.